core/misc/dropbutton.js

```diff
@@ -84,6 +84,9 @@ function DropButton (dropbutton, settings) {
   else {
     this.$dropbutton.addClass('dropbutton-single');
   }
+
+  // Fix parent element min-width, like <td class="operations">
+  this.$dropbutton.parent().css('min-width', this.$dropbutton.find('.dropbutton-widget').outerWidth());
```

Comment:
`.dropbutton-widget` is defined in the theme layer, so maybe using `this.$dropbutton.children().eq(0).outerWidth()` will be more robust ? also, I could add a test that this is not empty before trying to get outerWidth.

Reply:
I think the robustness of this is fine. Depending on the class is likely a better solution than depending on the structure of the markup (i.e. using `children()`). As an alternative, we could move this code into Seven itself, which would eliminate any concerns about markup structure. See https://github.com/backdrop/backdrop-issues/issues/2806#issuecomment-327976711.